### PR TITLE
Update repos.yml

### PR DESCRIPTION
**Pull request by ArchieScript**

### CODE REVIEW

File: config/repos.yml [hunk 1]
```diff
@@ -40,6 +40,9 @@ repos:
     index: 'https://github.com/X-Raym/REAPER-ReaScripts/raw/master/index.xml'
     default: true
     disporder: 12
+  -
+    link: 'https://github.com/ArchieScript/Archie_ReaScripts'
+    index: 'https://raw.githubusercontent.com/ArchieScript/Archie_ReaScripts/master/index.xml'
```
Comment (ArchieScript) [inline config/repos.yml]: Greetings! Changed the URL of the index as you said. It seems everything is properly done.

File: config/repos.yml [hunk 2]
```diff
@@ -115,3 +115,7 @@ repos:
   -
     link: 'https://github.com/IXix/JSFX'
     index: 'https://github.com/IXix/JSFX/raw/master/index.xml'
+  -   
+    link: 'https://github.com/ArchieScript/Archie_ReaScripts'
+    index: 'https://github.com/ArchieScript/Archie_ReaScripts/raw/master/index.xml' 
+  -
```
Comment (cfillion) [inline config/repos.yml]: Thank you, all good except for the extra hyphen in the last line. (It adds a null element to the repository list as per YAML syntax.)
```suggestion
```

**Comment by cfillion** [comment]
Thank you!

**Comment by ArchieScript** [comment]
And thank you!

**Pull request merged**